wizard-frontend/src/mirrord-schema.d.ts

```diff
@@ -0,0 +1,1800 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
```

Comment:
The disclaimer michal wish we had for the protobuf auto-generated code in our backend.